cmd/root/auth_test.go

```diff
@@ -181,3 +182,54 @@ func TestWorkspaceClientOrPrompt(t *testing.T) {
 		})
 	})
 }
+
+func TestMustAccountClientWorksWithDatabricksCfg(t *testing.T) {
+	testutil.CleanupEnvironment(t)
+
+	dir := t.TempDir()
+	configFile := filepath.Join(dir, ".databrickscfg")
+	err := os.WriteFile(
+		configFile,
+		[]byte(`
+			[account-1111]
+			host = https://accounts.azuredatabricks.net/
+			account_id = 1111
+			token = foobar
+			`),
+		0755)
+	require.NoError(t, err)
+
+	ctx, _ := cmdio.SetupTest(context.Background())
```

Comment:
This should also take the `cancel` output variable and run `t.Cleanup(cancel)` to clean it up.